verilog/formatting/formatter_test.cc

```diff
@@ -5499,8 +5498,7 @@ static constexpr FormatterTestCase kFormatterTestCases[] = {
         "      int count;\n"
         "      if (cfg.enable) begin\n"
         "        count = 1;\n"
-        "      end,\n"
-        "      utils_pkg::decrement())\n"
+        "      end, utils_pkg::decrement())\n"
```

Comment:
The previous version might be clearer, depending on whom you ask.  It is a weird case with unlexed code being a macro call argument.

Reply:
You are right. Generally argument N+1 should always start a new line when argument N is multiline. A more common case when it happens is just a long argument (some arithmetic expression, function call, etc) that has been wrapped.
This is something I plan to implement in the future. So far its just a TODO comment in the code. I'll make GH issues for such problems soon.

Reply:
Done https://github.com/chipsalliance/verible/issues/1211